.github/workflows/release-publish.yml

```diff
@@ -84,3 +94,28 @@ jobs:
           script: |
             cd /service/deploy/fosslight/
             bash /service/deploy/fosslight/deploy.sh
+
+  deploy_image:
```

Comment:
Combine jobs(`release-publish.yml` / `deploy-image-to-docker-hub.yml`) into one workflow